### DISCUSSION TITLE
Mojolicious shutdown itself

### DISCUSSION THREAD

**Discussion by kamer2004** [mojolicious/mojo#1954] [Q&A]
Hi,

I have a mojolicious application which runs behind a apache2 reverse proxy (for the ssl), on a debian virtual machine.

Sometimes, this mojolicious application shutdown, it disappears from 'ps -aux'. I have to restart it (with '/myapp.pl daemon -m production &' ; through ssh).

I would like to investigate the reason of this shutdown but I don't know how to do that ; has please someone an advice ?

Thanks  :)

**Comment by jhthorsen** [comment]
I would start it from systemd. That way the output will get logged to journal.
